Eatery Blue/Eatery Blue/Testing/DummyData.swift

```diff
@@ -68,5 +70,120 @@ enum DummyData {
             )
         ]
     )
+    
+    static let terraceMenu = Menu(categories: [
+        MenuCategory(category: "Pho Bar", items: [
+            MenuItem(
+                description: "A pho bowl with your choice of toppings.",
+                healthy: false,
+                name: "Pho Bowl",
+                price: 1025
+            ),
+            MenuItem(
+                description: "A rice bowl with your choice of toppings.",
+                healthy: false,
+                name: "Rice Bowl",
+                price: 1025
+            ),
+        ]),
+        MenuCategory(category: "Salads", items: [
+            MenuItem(
+                description: "A mixed salad with your choice of toppings.",
+                healthy: true,
+                name: "Mixed Salad",
+                price: 1100
+            )
+        ])
+    ])
+
+    static let terrace = Eatery(
+        campusArea: "Central",
+        events: (0..<5).map { offset in
+            Event(
+                canonicalDay: Day().advanced(by: offset),
+                endTimestamp: Day().advanced(by: offset).date(hour: 15, minute: 0).timeIntervalSince1970,
+                menu: terraceMenu,
+                startTimestamp: Day().advanced(by: offset).date(hour: 8, minute: 0).timeIntervalSince1970
+            )
+        },
+        id: 1002,
+        imageUrl: nil,
```

Comment:
Might want to consider adding the image to test how that interacts. Images often are problematic.